### ISSUE TITLE
PostgreSQL issue in migrations

### ISSUE THREAD

**Issue by jhermsen** [simpleidserver/SimpleIdServer#791]
Found issue using PostgreSQL in 5.0.1.

IdServer:
PostgreSQL migrations for DistributedCacheConfiguration don't seem to do anything. Starting IdServer shows no errors in log, but site appears blank. Changed to Redis and was able to open IdServer and login.

**Comment by simpleidserver** [comment]
"Hello, and sorry for my late reply! :)

This issue will be fixed in the 5.0.2 release.

Best regards,
SID

**Comment by simpleidserver** [comment]
Hello @jhermsen ,

I believe there is an issue because the table "public"."DistributedCache" is not created when the PostgreSQL database is selected.

I made some changes to the "Program.cs" file to fix this. Could you please execute the following SQL query and try again?

```
CREATE TABLE IF NOT EXISTS "public"."DistributedCache"
"Id" text COLLATE pg_catalog."default" NOT NULL,
"Value" bytea,
"ExpiresAtTime" timestamp with time zone,
"SlidingExpirationInSeconds" double precision,
"AbsoluteExpiration" timestamp with time zone,
CONSTRAINT "DistCache_pkey" PRIMARY KEY ("Id")
)
```

KR,

SID
